js/trbinance.js

```diff
@@ -0,0 +1,1062 @@
+                'rate': feeCurrency,
+            },
+        };
+    }
+
+    async fetchWithdrawals (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        // const request = {};
+        // const response = await this.privateGetwithdraws (this.extend (request, params));
+        // const withdrawals = this.safeValue (response, 'withdrawals');
+        const currency = undefined;
+        const response = [
+            {
+                'id': 1,
+                'clientId': '1',
+                'asset': 'BTC',
+                'network': 'BTC',
+                'address': '1G58aoKLVd1vHkv7wi6R2rKUrjuk4ZRtY3',
+                'amount': '0.001',
+                'fee': '0.0005',
+                'txId': '',
+                'status': 4,
+                'createTime': 1572359825000,
+            },
+        ];
+        return this.parseTransactions (response, currency, since, limit);
+    }
+
+    async fetchDeposits (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const currency = undefined;
+        // const request = {};
+        // const response = await this.privateGetDeposits (this.extend (request, params));
+        // const deposits = this.safeValue (response, 'deposits');
+        const response = [
+            {
+                'id': 1,
+                'asset': 'BTC',
+                'network': 'BTC',
+                'address': '2',
+                'addressTag': '2',
+                'txId': '1',
+                'amount': '1.000000000000000000000000000000',
+                'status': 1,
+                'insertTime': '0',
+            },
+        ];
+        return this.parseTransactions (response, currency, since, limit);
+    }
+
+    async fetchDepositAddress (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        // const currency = undefined;
+        // const request = {};
+        // const response = await this.privateGetDepositAddress (this.extend (request, params));
+        // const deposits = this.safeValue (response, 'deposits');
+        const response = {
+            'address': '0x6915f16f8791d0a1cc2bf47c13a6b2a92000504b',
+            'addressTag': '1231212',
+            'asset': 'BNB',
+        };
+        // return this.parseTransactions (response, currency, since, limit);
+        return response;
+    }
+
+    async createOrder (symbol, type, side, quantity = undefined, price = undefined, stopPrice = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' createOrder() requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        side = (side === 'buy') ? '1' : '0';
+        const request = {
+            'symbol': symbol,
+            'side': side,
+            'type': type.toString (),
+        };
+        if (type === 1) {
```

Comment:
The `createOrder()` type parameter must be either `market` or ` limit` as described in the Manual here:

- https://github.com/ccxt/ccxt/wiki/Manual#placing-orders

Also, let's add the proper support for `clientId` and `clientOrderId` here.
In general, if the exchange has the support for client order ids, we must support 2 params-overrides, the exchange specific param `clientId` and the unified param `clientOrderId`. See how that is done in the other exchanges.

The same is true for the `stopPrice` param – see how it is done in the other exchanges. If `stopPrice` is specified, then it is a stop order (https://www.trbinance.com/apidocs/#public-api-definitions).